vue-thacer/src/components/TheCeramicText.vue

```diff
@@ -4,8 +4,18 @@
     <p class="lead text-muted m-0">{{ ceramInv }}</p>
     <ul>
       <div v-for="(value, name, index) in ceramData" :key="index">
-        <div v-if="value && name != 'ID' && name != 'Pi' && name != 'x' && name != 'y'">
-          {{ name }}
+        <div
+          v-if="
+            value &&
+            name != 'ID' &&
+            name != 'Pi' &&
+            name != 'x' &&
+            name != 'y' &&
+            name != 'secteur_ID'
```

Comment:
`!==` :)
We should have it in the linter ! 

If `value` and `name` are easily renamed to something which describe better what they contains, don't hesitate